src/site/constants/drupals.js

```diff
@@ -37,7 +37,7 @@ const ENABLED_ENVIRONMENTS = new Set([
 ]);
 
 const PREFIXED_ENVIRONMENTS = new Set([
-  // ENVIRONMENTS.LOCALHOST,
+  ENVIRONMENTS.LOCALHOST,
```

Comment:
Without the prefix on localhost, content team members can't preview their changes in Heroku on vagov-content.